### DISCUSSION TITLE
Support for Apple M1 / M2 / M3 CPUs - Metal/MPS

### DISCUSSION THREAD

**Discussion by ChristianWeyer** [microsoft/onnxruntime-genai#366] [Support for Targets (OS / EPs / Hardware)]
Hey there,

it would be great to have Metal as a supported target to fully leverage the power of the M1, M2, M3 chip family.
Thanks!

**Comment by natke** [comment]
Thank you @ChristianWeyer! I'm moving this issue into our Discussions forum, so that we can prioritize.
